.github/workflows/merge_conan_build.yml

```diff
@@ -6,6 +6,7 @@ on:
     branches:
       - master
       - stable/v3.x
+      - thin_provisioning
```

Comment:
This needs to go into the thin_provisioning branch prior to this PR being able to build. So you may need to direct push or have another pr that you just blindly merge first.